sandbox/namespace_linux.go

```diff
@@ -55,9 +55,7 @@ func createBasePath() {
 }
 
 func removeUnusedPaths() {
-	for {
-		time.Sleep(time.Duration(gpmCleanupPeriod))
-
+	for range time.Tick(time.Duration(gpmCleanupPeriod)) {
```

Comment:
when I had it multiplying two numbers it did complain. But just a single value it doesn't seem to complain without the type cast. removing it